### ISSUE TITLE
Issue in the example on .../keras/basic_classification page

### ISSUE THREAD

**Issue by jjimenezcapi** [tensorflow/tensorflow#32134]
The code below and explanation  pick up from your page https://www.tensorflow.org/tutorials/keras/basic_classification its no actually working. In your case is working for the image in the position 0, but no for the rest due to always is picking up/checking test_labels[i] where i in this case is 0, so always is comparing with test_labels[0] which is '9' (anckle boot).
If you try with test_image[1] , or test_image[2] , ..... you can check how the chart is no show the correct color (predicted/true label)

```
img = test_images[0]

predictions_single = model.predict(img)

plot_value_array(0, predictions_single, test_labels)
plt.xticks(range(10), class_names, rotation=45)
plt.show()
```

I only register this issue in order you can improve your documentation. 
Thanks for you software.

**Comment by gadagashwini** [comment]
@jjimenezcapi, In the above mentioned tutorial, test_labels[i], here i is the position of image in the dataset. Where, i=0,  ankle boot image and i=1 T-shirt/top image. Is it allowed to choose any of the image from dataset. Thanks!

**Comment by jjimenezcapi** [comment]
Yes I know, but the issue is about the function plot_value_array
Its no possible to call it with another value that 0 for single predictions. It raised and unbound exception . Its only FYI

**Comment by jvishnuvardhan** [comment]
@jjimenezcapi Thanks for pointing this issue. Actually it was corrected recently, but the website is not updated. You could check the [source code](https://github.com/tensorflow/docs/blob/r2.0rc/site/en/tutorials/keras/basic_classification.ipynb) and [google colab](https://colab.sandbox.google.com/github/tensorflow/docs/blob/r2.0rc/site/en/tutorials/keras/basic_classification.ipynb#scrollTo=YFc2HbEVCaXd), both were updated and waiting for website to get update.

As you mentioned, In the website it shows
```
plot_value_array(0, predictions_single, test_labels)
```
In the source codes, it was updated and shows as

```
plot_value_array(1, predictions_single[0], test_labels)
```

I am closing this issue as it was resolved. Thanks for your contribution.
